portal_tables/sync_tools.py

```diff
@@ -36,6 +36,9 @@ def add_missing_info(tools: pd.DataFrame, grants: pd.DataFrame) -> pd.DataFrame:
                 synapse_links.append("".join(["[Link](", s , ")"]))
         tools.at[_, "synapseLink"] = ", ".join(set(synapse_links))
         
+        if tools.at[_,"PublicationViewKey"] == "":
+            tools.at[_,"PublicationViewKey"] = "Pending Annotation"
```

Comment:
Maybe use the `or` operator here to assign a default value instead?

```suggestion
        tools.at[_,"PublicationViewKey"] = tools.at[_,"PublicationViewKey"] or "Pending Annotation"
```